src/tests/public_api/test_helpers/proxy.h

```diff
@@ -180,26 +306,33 @@ class Proxy : public core::Thread, private packet::IWriter {
         if (pp->udp()->dst_addr == recv_source_config_.bind_address) {
             pp->udp()->dst_addr = receiver_source_endp_;
             LONGS_EQUAL(status::StatusOK, source_queue_.write(pp));
+        } else if (pp->udp()->dst_addr == recv_control_config_.bind_address) {
+            pp->udp()->dst_addr = receiver_control_endp_;
+            LONGS_EQUAL(status::StatusOK, control_queue_.write(pp));
         } else {
             pp->udp()->dst_addr = receiver_repair_endp_;
             LONGS_EQUAL(status::StatusOK, repair_queue_.write(pp));
         }
 
         for (;;) {
-            const size_t block_pos = pos_ % (n_source_packets_ + n_repair_packets_);
+            const size_t block_pos =
+                pos_ % (n_source_packets_ + n_repair_packets_ + n_control_packets_);
```

Comment:
As mentioned above, `n_control_packets_` doesn't make sense, let's remove it. We can just never drop control packets for now, if we'll ever need it, we can add separate flag for that.